figures/README.md

```diff
@@ -40,6 +40,7 @@ However, we list information about the resources, intermediate steps, and [PBTA
 
 | Figure | Individual script | Notes on requirements | Linked analysis modules | PBTA data files consumed |
 |--------|--------|------------------|-------------------------|-----------------------------|
+| Figure 1 | [scripts/fig1-sample-distribution.R`](./scripts/fig1-sample-distribution.R) | NA | [`sample-distribution-analysis`](https://github.com/AlexsLemonade/OpenPBTA-analysis/tree/master/analyses/sample-distribution-analysis) |  `pbta-histologies.tsv` |
```

Comment:
Perhaps instead of NA just put that it doesn't have high RAM requirements or if there's anything else special about it. I think that's preferred to putting NA so people don't think you just forgot to fill it out.

Reply:
Ah very true 👍